docs/resources/cts_tracker_v1.md

```diff
@@ -27,6 +26,8 @@ The following arguments are supported:
 * `bucket_name` - (Required) The OBS bucket name for a tracker.
 
 * `file_prefix_name` - (Optional) The prefix of a log that needs to be stored in an OBS bucket.
+The value can contain letters, digits, and special characters `.-_`, but cannot contain spaces.
+The length is 1 to 64 characters.
```

Comment:
please add two spaces at the beginning of the line.

Reply:
I have added two spaces at the beginning